### PR TITLE
feat: add multi themes support

### PR DESCRIPTION
**Pull request by unimu-cic**

### CODE REVIEW

File: src/components/DiagramFrame/SeqDiagram/MessageLayer/Block/Statement/Fragment/FragmentTryCatchFinally.vue [hunk 1]
```diff
@@ -27,8 +27,8 @@
         </block>
       </div>
       <template v-for="(catchBlock, index) in tcf.catchBlock()" :key="index + 500">
-        <div class="segment mt-2 border-t border-solid">
-          <div class="header text-skin-fragment" :key="index + 1000">
+        <div class="segment text-xs mt-2 border-t border-solid">
+          <div class="header inline-block text-skin-participant" :key="index + 1000">
```
Comment (MrCoder) [inline src/components/DiagramFrame/SeqDiagram/MessageLayer/Block/Statement/Fragment/FragmentTryCatchFinally.vue:31]: why `text-skin-participant`?

Comment (MrCoder) [reply src/components/DiagramFrame/SeqDiagram/MessageLayer/Block/Statement/Fragment/FragmentTryCatchFinally.vue:31]: Segment text size is not consistent.

**Review by MrCoder — CHANGES_REQUESTED** [review]

File: src/components/DiagramFrame/SeqDiagram/MessageLayer/Block/Statement/Fragment/FragmentTryCatchFinally.vue (hunk 1 above)
Comment (unimu-cic) [reply src/components/DiagramFrame/SeqDiagram/MessageLayer/Block/Statement/Fragment/FragmentTryCatchFinally.vue]: > why `text-skin-participant`?

Removed

File: index.html [hunk 2]
```diff
@@ -76,6 +76,10 @@
         <pre class="zenuml"></pre>
       </div>
     </div>
+    <!--  headlessui protal root hack: https://github.com/tailwindlabs/headlessui/discussions/666#discussioncomment-4966117  -->
```
Comment (MrCoder) [inline index.html:79]: This adds a big burden to the client to use this library. For example, all the demo pages under `/cy` folder needs to be updated. Mermaid integration would also need to be updated. I am worried that if this design allows multiple ZenUML diagrams to be rendered on one page. Mermaid demo pages is such a case.

If I understand it correctly, all "clients" would need to have such an element for the choose theme to work. It breaks the API of:
1. const zenuml = new ZenUML(element)
2. zenuml.render(code, theme)
We need to find another way (maybe even hackier) to implement this. How about create this node at the begining?

**Review by MrCoder — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (MrCoder) [reply index.html]: It still does not work on my computer. See the video below:

https://github.com/mermaid-js/zenuml-core/assets/169417/8ab63c67-a2fe-4871-b7bd-a87b8f6f8d72

Comment (MrCoder) [reply index.html]: It seems that it has not been resolved.

**Pull request merged**